src/libs/actions/CloseAccount.js

```diff
@@ -2,13 +2,6 @@ import Onyx from 'react-native-onyx';
 import ONYXKEYS from '../../ONYXKEYS';
 import CONST from '../../CONST';
 
-/**
- * Clear CloseAccount error message to hide modal
- */
-function clearError() {
```

Comment:
hmm you might be right, I noticed at one point in testing but then it went away after I'd reloaded and then I couldn't reproduce. Let me try again.

Reply:
Though don't think this method would actually stop the loader from showing since I think that is controlled by the `isLoading` key that's defined here: https://github.com/Expensify/App/blob/63f314c054d82ad0ee5e59ec5349d9b35d656bea/src/CONST.js#L730

Reply:
The more I noodle on this I think maybe we shouldn't be trying to address this here?

Really these props are all reacted-to by the `Form` component, this component is sort of just attempting to pass that data down along to it.

The only way to really solve it is to use the `useEffect` / lifecycle methods that were being used but those don't really make sense to add here since our component doesn't actually depend on them.

If we were to re-implement the lifecycle methods here we'd need `useEffect` which is just going to set some onyx data to the default because that's what the `<Form>` component will react to.